### DISCUSSION TITLE
cannot install any plugins . failed with check the permissions

### DISCUSSION THREAD

**Discussion by jaheshkhan** [netbox-community/netbox#6551] [General]
today i tried to install fresh install of netbox. so now the netbox version is 2.11.6

python version is 3.8.5

Then i tried to install plugins like 
1. nextbox-ui-plugin, 
2. ntc-netbox-plugin-onboarding , 
3. netbox-topology-views.

none of the above plugin installed all ended withe permission issue as below

"ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/opt/netbox/venv/lib/python3.8/site-packages/nextbox_ui_plugin'
Check the permissions."

Installing collected packages: zipp, ntc-netbox-plugin-onboarding
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/opt/netbox/venv/lib/python3.8/site-packages/zipp.py'
Check the permissions.

Installing collected packages: netbox-topology-views
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/opt/netbox/venv/lib/python3.8/site-packages/netbox_topology_views'
Check the permissions.

I didnt understand what permission im missing

The steps followed are as follows:
$ source /opt/netbox/venv/bin/activate
(venv) $ pip3 install nextbox-ui-plugin

**Comment by zircon** [comment]
`/opt/netbox` is owned by `root:root`, which means you won't have write access to it as a standard user.

Try this instead:
```
sudo -i
source /opt/netbox/venv/bin/activate
pip3 install nextbox-ui-plugin
```

Then add your additional packages to `/opt/netbox/local_requirements.txt` so that they're reinstalled when you next upgrade NetBox.
